cxx/structureTranslationModule/translator/NrelFromNodeTranslator.cpp

```diff
@@ -0,0 +1,50 @@
+#include "NrelFromNodeTranslator.hpp"
+
+namespace structureTranslationModule
+{
+NrelFromNodeTranslator::NrelFromNodeTranslator(
+    ScMemoryContext * context)
+  : StructureTranslator(context)
+{
+}
+
+std::vector<std::string> NrelFromNodeTranslator::translate(ScAddr const & structAddr) const
+{
+  std::vector<std::string> translations;
+  std::string const NODE_ALIAS = "_node";
+  std::string const SOURCE_ALIAS = "_source";
+  std::string const NREL_ALIAS = "nrel";
+  std::string const EDGE_ALIAS = "_edge";
```

Comment:
move this constants to the separate class, make it static and use in all the translators. It is a good memory optimisation